i3lock.c

```diff
@@ -1558,6 +1660,25 @@ int main(int argc, char *argv[]) {
         {"pass-power-keys", no_argument, NULL, 603},
         {"pass-volume-keys", no_argument, NULL, 604},
 
+        // custom commands for pass keys
```

Comment:
Hmmm in this case maybe make this 610 and the following blocks 62x, 63x, etc?

Reply:
I've amended it. How about now?